### ISSUE TITLE
There are duplicate entries for the same mod but from different sources 

### ISSUE THREAD

**Issue by T3sT3ro** [gorilla-devs/ferium#329]
### Description

I have some mods which include Roughly Enough Items as their required dependency. The problem is, that one mod is referenced from the modrinth site and the other from the curseforge. Removing one of the REI mods adds it to the dependency list once again and prints 

> Warning: 1 duplicate files were found RoughlyEnoughItems-9.1.595.jar. Remove the mod it belongs to

I am afraid that this doesn't have a simple fix, until unless the #140 compare-by-hash is not implemented

### Steps to reproduce

Have two mods depend on REI. One from curseforge, one from modrinth.

There is no dependency graph, so it's rather hard to see which two mods depend on REI. If need arises, I can try to bruteforce it.

### Operating System

Linux

### Are you using the latest version of ferium?

- [X] I am using the latest version of ferium

**Comment by theRookieCoder** [comment]
> unless the #140 compare-by-hash is implemented

#140 won't help with adding the same mod from different sources.

> Removing one of the REI mods adds it to the dependency list

This should fix the issue, I guess you didn't use the remove command correctly?

**Comment by T3sT3ro** [comment]
> https://github.com/gorilla-devs/ferium/issues/140 won't help with adding the same mod from different sources.

If two different mods were compared on hashes after their download, then those two mods could be unified into or one could be marked as a primary version of the other in the profile. Something like selecting appropriate package on linux from different PPAs

> This should fix the issue, I guess you didn't use the remove command correctly?

I've checked history and indeed, turns out instead of removing Roughly Enough Items (duplicate) I removed Roughly Enough Resources. I've tested right now removing the duplicate mod by ID and it indeed works as intended.
